src/include/kdbmerge.h

```diff
@@ -10,11 +10,12 @@
 #define KDBMERGE_H_
 
 #include "kdb.h"
-#include "kdberrors.h"
 #include "kdbprivate.h"
 
 #ifdef __cplusplus
 extern "C" {
+using Key = ckdb::Key;
```

Comment:
to fix your problem, put elektraMerge also under the namespace ckdb.

Reply:
Thank you for the hint! Fixed in #3234